core/domain/config_domain.py

```diff
@@ -70,6 +71,15 @@
             'schema': {
                 'type': 'unicode'
             }
+        }, {
+            'name': 'url_fragment',
```

Comment:
I noticed the structure for this property has changed. Will that cause any issues? (Test by checking out develop, setting something there, switching to this branch, seeing if the existing stuff that uses that property works fine before editing it, then editing the config property, and seeing if the stuff that uses that property works fine after editing it.)